src/pages/preferences/preferencesTabs/NotificationsTab.tsx

```diff
@@ -89,6 +93,12 @@ export const NotificationsTab: React.VFC = () => {
           value={slackUsernameField}
           data-cy="slack-username-field"
         />
+        <StyledTextInput
+          label="Slack Member Id"
+          onChange={handleFieldUpdate(setSlackMemberIdField)}
+          value={slackMemberIdField}
```

Comment:
Maybe we can add a `description` props that describes how to get your member ID from Slack